application/src/main/java/org/thingsboard/server/controller/BaseController.java

```diff
@@ -458,11 +458,15 @@ protected SecurityUser getCurrentUser() throws ThingsboardException {
     }
 
     Tenant checkTenantId(TenantId tenantId, Operation operation) throws ThingsboardException {
+        return checkTenantId(tenantId, operation, getCurrentUser());
```

Comment:
Please revert all these changes and better recreate the PR without those changes in the commit history. This is completely not needed.